### PR TITLE
Massive Cleanup

### PR DESCRIPTION
**Pull request by SticksDeveloper**
- Removed
Everything related to contagion.
Props
Some Entities which relate to removing everything related to contagion

### CODE REVIEW

**Comment by Paneedah** [comment]
Willing to merge this. Requesting review by @Desoroxxx.

File: src/main/java/com/paneedah/mwc/ModernWarfareMod.java [hunk 1]
```diff
@@ -12,13 +12,15 @@
...
 import net.minecraft.creativetab.CreativeTabs;
+import net.minecraftforge.event.entity.EntityJoinWorldEvent;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.Mod.EventHandler;
 import net.minecraftforge.fml.common.SidedProxy;
 import net.minecraftforge.fml.common.event.FMLInitializationEvent;
 import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
 import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
 import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+import net.minecraftforge.fml.common.gameevent.PlayerEvent;
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/ModernWarfareMod.java:23]: Why two new imports when nothing in the file got added?

File: src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java [hunk 2]
```diff
@@ -5,6 +5,10 @@
 
 import com.paneedah.mwc.ModernWarfareMod;
 import com.paneedah.mwc.models.*;
+import com.paneedah.mwc.models.AACHoneyBadgerSilencer;
+import com.paneedah.mwc.models.Acog2;
+import com.paneedah.mwc.models.AcogReticle;
+import com.paneedah.mwc.models.AcogScope2;
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java:11]: Again what's up with the imports

File: src/main/java/com/paneedah/mwc/proxies/CommonProxy.java [hunk 3]
```diff
@@ -124,11 +123,10 @@ public void init(final ModernWarfareMod mod, final CompatibleFmlInitializationEv
         ModernWarfareMod.MOD_CONTEXT.init(mod);
 
         Entities.init(this);
-        Vehicles.init(this);
```
Comment (Desoroxxx) [inline src/main/java/com/paneedah/mwc/proxies/CommonProxy.java:127]: ?

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

**Comment by Desoroxxx** [comment]
This is too messy.

You are trying to do five different things at once:

- Cleanup
- Remove Contagion
- And remove vehicles?????

Also why are a dozen random imports added for no reason?

Please reopen one pr with one goal not one with three goals, once these issues are fixed

File: src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java (hunk 2 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/items/guns/AACHoneyBadgerFactory.java:11]: IDK It just happened the second I moved a file by accident

File: src/main/java/com/paneedah/mwc/ModernWarfareMod.java (hunk 1 above)
Comment (SticksDeveloper) [reply src/main/java/com/paneedah/mwc/ModernWarfareMod.java:23]: I'll revert It no reason why they got added just some random ones

**Pull request closed without merging**